aws_xray_sdk/ext/django/middleware.py

```diff
@@ -50,9 +59,10 @@ def __call__(self, request):
             recorder=xray_recorder,
             sampling_req=sampling_req,
         )
-
         if self.in_lambda_ctx:
-            segment = xray_recorder.begin_subsegment(name)
+            segment = xray_recorder.begin_subsegment(name, namespace="remote")
```

Comment:
Can you please clarify the reason for setting `namespace="remote"` here? As far as I know, setting a subsegment's namespace as remote manually makes it show on the service map and should not affect the ability to search annotations.

Reply:
Helpfully directly from the Xray team here:

> change the namespace of the subsegments from "local" to "remote" and to then use the service filter. When the namespace is set to "remote", Xray will infer a node base which can be searched up with the service filter.

So this enables service filter use in this case too

Reply:
I referred back to our team and confirmed that we do index all annotations in a trace, which includes annotations added on subsegments. In this case, it doesn't seem necessary to set `namespace="remote"` in order to be able to search annotations, so we can merge this PR once that adjustment is made!

Reply:
Thank you!   Updated with removing this.    Will plan to remove our patches once this is released, and looking forward to nobody else hitting this issue!